setup.py

```diff
@@ -22,7 +22,7 @@ def read(fname):
 
 setup(
     name='ecs-cluster',
-    version='1.0.6',
+    version='1.0.7',
```

Comment:
should this be `1.1.0`?

Reply:
I don't have a strong preference, but I can change it up